pyodide_build/tests/test_config.py

```diff
@@ -216,3 +217,112 @@ def test_cli_config_subset():
         assert value in BUILD_VAR_TO_KEY, (
             f"All cli config values should be in build var to key mapping, but {value} is not"
         )
+
+
+class TestParseMakefileEnvs:
```

Comment:
How about a test with an assignment like `A=2` that doesn't start with export? Also, `export A=2 # this is a comment` isn't handled though it seems to set `A` to `"2 "`.